### PR TITLE
Brianbolt/issue879

### LINKED ISSUE #879: Purity operator should be null by default in single reg

### PR DESCRIPTION
**Pull request by brianbolt**
Fix requires equivalent updates to Lot_Custom.js on any acas_custom that has it.

Fixes #879

### CODE REVIEW

File: modules/CmpdReg/src/client/custom/Lot_Custom.js [hunk 1]
```diff
@@ -498,6 +507,9 @@ $(function() {
               if (this.purityMeasuredByCodeController.getSelectedModel().isNew()) purityMeasuredBy = null;
               else purityMeasuredBy = this.purityMeasuredByCodeController.getSelectedModel();
                 var vendor;
+              if (this.operatorCodeController.getSelectedModel().isNew()) purityOperator = null;
+              else purityOperator = this.operatorCodeController.getSelectedModel();
+                var tareWeightUnits;
```
Comment (bffrost) [inline modules/CmpdReg/src/client/custom/Lot_Custom.js:512]: Weird indentation here for `var tareWeightUnits` and then it is not used directly below. Was there supposed to be a `var purityOperator` above here?

**Review by bffrost — CHANGES_REQUESTED** [review]
Spotted one section that looks off

File: modules/CmpdReg/src/client/custom/Lot_Custom.js (hunk 1 above)
Comment (brianbolt) [reply modules/CmpdReg/src/client/custom/Lot_Custom.js]: Thanks, fixed this.

**Review by bffrost — APPROVED** [review]
LGTM!

**Pull request merged**